program animations

shuuvi opened the pull request:
https://github.com/user-attachments/assets/5940edf1-9f8a-4575-af69-2fe91b3d4808

added the animation to the what we offer page but didn't add to the header because I'm not sure if I was supposed to.

shuuvi committed:
added animations

stanleylew5 reviewed (approved):
i fuck w the animation, can u solve ur merge conflicts and also have the animations be defined like andres did here. https://github.com/acm-ucr/acm-atlas/pull/1036/files
his version is more clean than what I have in the slides

stanleylew5 reviewed (changes requested):
whoops meant to request changes

shuuvi committed:
some conflicts

shuuvi committed:
finished requested changes and merge conflicts

shuuvi committed:
fixed merge conflicts

shuuvi commented:
changed the header for the animation and fixed merge conflicts :D

stanleylew5 committed:
Merge branch 'dev' into shuuvi/programAnimation

Signed-off-by: Stanley Lew <135681592+stanleylew5@users.noreply.github.com>

stanleylew5 committed:
fix centering

stanleylew5 reviewed (changes requested):
![image](https://github.com/user-attachments/assets/24262623-56ba-456f-a8e1-bad7364f0143)
![image](https://github.com/user-attachments/assets/916d29b0-c921-4b46-97e3-da5260381d5a)

Have your animations be defined inside your motion divs like this

shuuvi committed:
conflicts

shuuvi committed:
fixed requested changes for defining animation

shuuvi committed:
fixed push conflicts

shuuvi commented:
finished requested changes, implemented the varient tag into animation block to help condense and added another drop animation that is slightly faster.
